builder/proxmox/common/step_finalize_template_config.go

```diff
@@ -105,6 +99,17 @@ func (s *stepFinalizeTemplateConfig) Run(ctx context.Context, state multistep.St
 		}
 	}
 
+	// Disks that get replaced by the builder end up as unused disks -
+	// find and remove them.
+	rxUnused := regexp.MustCompile(`^(unused)\d+`)
```

Comment:
Unless you're using the regexp to extract things, I'd think the parentheses are not useful here